haigha/transports/socket_transport.py

```diff
@@ -4,12 +4,17 @@
 https://github.com/agoragames/haigha/blob/master/LICENSE.txt
 '''
 
+from haigha.exceptions import ConnectionError
 from haigha.transports.transport import Transport
 
 import errno
+import logging
 import socket
 
 
+LOG = logging.getLogger(__name__)
```

Comment:
We don't want to create a new logger here. Haigha has a system in place for configuring the logger for the connection and everything that it encompasses. The logger is either passed by the user to Connection or Connection will use its own default if one wasn't passed by the user. After that, Connection expects **that** logger to be used by the channels and transport. By creating a new logger here, we're interfering with what the user and/or Connection intended. See references to `self.connection.logger` in my example haigha PR https://github.com/agoragames/haigha/pull/80/files. You will also find references to `self.connection.logger` in `SocketTransport.read`, etc.